content/backmatter.tex

```diff
@@ -522,6 +522,25 @@ \section{Version 1.5}
 \CONST{SHMEM\_MALLOC\_ATOMICS\_REMOTE} and \CONST{SHMEM\_MALLOC\_SIGNAL\_REMOTE}.
 \\ See Section \ref{subsec:shmmallochint} and \ref{subsec:library_constants}.
 %
+\item Changed the broadcast function to update the dest object on all PEs,
+including the root PE.
+\\ See Section \ref{subsec:shmem_broadcast}.
```

Comment:
This change is only for team-based broadcasts, not active-set based broadcasts. Can you please clarify?